charts/galoy/values.yaml

```diff
@@ -690,47 +753,40 @@ kratos:
           code:
             enabled: true
             config:
-              # Defines how long the verification or the recovery code is valid for (default 1h)
               lifespan: 15m
         flows:
           error:
-            ui_url: http://127.0.0.1:4002/error
+            ui_url: http://api:4002/error
           settings:
-            ui_url: http://127.0.0.1:4002/settings
+            ui_url: http://api:4002/settings
             privileged_session_max_age: 15m
             required_aal: aal1
             after:
               profile:
                 hooks:
                   - hook: web_hook
                     config:
-                      # url: http://e2e-tests:4002/auth/after_settings_hooks
                       url: http://invalid-because-we-dont-want-profile-to-be-updated
                       method: POST
-                      body: file:///home/ory/body.jsonnet
+                      body: base64://ZnVuY3Rpb24oY3R4KSB7CiAgICBpZGVudGl0eV9pZDogaWYgc3RkLm9iamVjdEhhcyhjdHgsICJpZGVudGl0eSIpIHRoZW4gY3R4LmlkZW50aXR5LmlkIGVsc2UgbnVsbCwKICAgIHBob25lOiBpZiBzdGQub2JqZWN0SGFzKGN0eC5pZGVudGl0eS50cmFpdHMsICJwaG9uZSIpIHRoZW4gY3R4LmlkZW50aXR5LnRyYWl0cy5waG9uZSBlbHNlIG51bGwsCiAgICBlbWFpbDogaWYgc3RkLm9iamVjdEhhcyhjdHguaWRlbnRpdHkudHJhaXRzLCAiZW1haWwiKSB0aGVuIGN0eC5pZGVudGl0eS50cmFpdHMuZW1haWwgZWxzZSBudWxsLAogICAgc2NoZW1hX2lkOiBjdHguaWRlbnRpdHkuc2NoZW1hX2lkLAogICAgZmxvd19pZDogY3R4LmZsb3cuaWQsCiAgICBmbG93X3R5cGU6IGN0eC5mbG93LnR5cGUKfQ==
```

Comment:
base64 representation of this file https://github.com/GaloyMoney/galoy/blob/8d46de4421ef4bc424ad160aff9ee169b1b418d1/dev/ory/body.jsonnet